### DISCUSSION TITLE
2.3.250 version: installation stuck at 95% for a long time then it completes with errors

### DISCUSSION THREAD

**Discussion by kikirizzi** [Security-Onion-Solutions/securityonion#10461] [General]
I'm having problems installing a fresh new infrastructure.
ISO was correctly downloaded and has passed succesfully all signature checks.
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/11602190/sosetup.log)

When I try to install it on VMware ESXi with 4vCPU/32GB RAM/250 GB Hard Disk or to a bare metal server with 6core/22 GB RAM/1TB Hard disk the installation goes fine, but logging in and trying to configure any role (EVAL, STANDALONE, DISTRIBUTED it doesn't matter) after inserting all the requested informations it stuck at 95% for a long time. After 25/30 minutes it completes with an error. The /root/error.log is empty (0 bytes).

Thx

**Comment by dougburks** [comment]
Looking at your sosetup.log, it looks like you're having network issues. Double-check your network, firewall, and proxy configuration. If you don't have full Internet access, you may need to do an airgap installation. Here is some documentation which might help:
https://docs.securityonion.net/en/2.3/getting-started.html
https://docs.securityonion.net/en/2.3/firewall.html
https://docs.securityonion.net/en/2.3/proxy.html
https://docs.securityonion.net/en/2.3/airgap.html

**Reply by kikirizzi** [reply]
Thank you for the answer. I've tried to reconfigure the firewall but after other some failures, I have decided to bypass the firewall completely.
I have assigned a public IP address to the box during network setup but the installation failed the same way.
It is not a home connectivity

I've asked the provider if they apply some filters but they told me that no filters are used.

Is it possible to download an older ISO to test it?

**Reply by dougburks** [reply]
Per my previous response, have you tried an airgap installation?
https://docs.securityonion.net/en/2.3/airgap.html

**Comment by kikirizzi** [comment]
Yes, Airgap installation brings me to the identical result.

**Reply by dougburks** [reply]
And just to double-check...did you choose the Airgap option here?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/162ebc9e-8cee-4290-a745-a1562cd3723b)

Are you able to share the sosetup.log from the airgap installation?

**Comment by kikirizzi** [comment]
Hi

Today I’ve tried another installation on VMware.
Attached you can find screenshoots and also log files.

I don’t know why this is happening, I’ve tried ISO, USB, DVD installation but I have always the same result.
Thank you for you patience


Tiziano




[PS Innovation Srl]
Ing. Tiziano Zanotti | Senior IT System Engineer
T: +39 030 64 90 141<tel:+390306490141> | M: +39 348 65 05 929<tel:+393486505929>
E: ***@***.******@***.***> | W: https://www.psinnovation.it<https://www.psinnovation.it/>
Address: Via Camillo Brozzoni, 9 - 25125 Brescia


Avviso di riservatezza: Il contenuto di questa e-mail e degli allegati può avere natura confidenziale ad uso esclusivo del destinatario effettivo. Se avete ricevuto questo messaggio per errore, Vi preghiamo di informarci restituendo l'e-mail all'indirizzo ***@***.******@***.***> e di procedere alla eliminazione dello stesso. Ogni utilizzo improprio è contrario ai principi del GDPR 679/2016. Grazie per la collaborazione.

Privacy notice: The content of this e-mail and attachments may be of a confidential nature for the exclusive use of the actual recipient. If you have received this message by mistake, please inform us by returning the e-mail to ***@***.******@***.***> and deleting it. Any improper use is contrary to the principles of GDPR 679/2016. Thanks for your collaboration

**Reply by dougburks** [reply]
I don't see any screenshots or log files.

**Comment by kikirizzi** [comment]
Hello Doug

This morning I’ve downloaded the latest release 2.3.260 and started a new installation.
Everything worked like a charm and now I’m up and running.

Thank you for your support


Tiziano


[PS Innovation Srl]
Ing. Tiziano Zanotti | Senior IT System Engineer
T: +39 030 64 90 141<tel:+390306490141> | M: +39 348 65 05 929<tel:+393486505929>
E: ***@***.******@***.***> | W: https://www.psinnovation.it<https://www.psinnovation.it/>
Address: Via Camillo Brozzoni, 9 - 25125 Brescia


Avviso di riservatezza: Il contenuto di questa e-mail e degli allegati può avere natura confidenziale ad uso esclusivo del destinatario effettivo. Se avete ricevuto questo messaggio per errore, Vi preghiamo di informarci restituendo l'e-mail all'indirizzo ***@***.******@***.***> e di procedere alla eliminazione dello stesso. Ogni utilizzo improprio è contrario ai principi del GDPR 679/2016. Grazie per la collaborazione.

Privacy notice: The content of this e-mail and attachments may be of a confidential nature for the exclusive use of the actual recipient. If you have received this message by mistake, please inform us by returning the e-mail to ***@***.******@***.***> and deleting it. Any improper use is contrary to the principles of GDPR 679/2016. Thanks for your collaboration
